js/apps/account-ui/src/account-security/LinkedAccounts.tsx

```diff
@@ -8,27 +8,41 @@ import { Page } from "../components/page/Page";
 import { usePromise } from "../utils/usePromise";
 import { AccountRow } from "./AccountRow";
 import { useEnvironment } from "@keycloak/keycloak-ui-shared";
+import { LinkedAccountsToolbar } from "../resources/LinkedAccountsToolbar";
 
 export const LinkedAccounts = () => {
   const { t } = useTranslation();
   const context = useEnvironment();
-  const [accounts, setAccounts] = useState<LinkedAccountRepresentation[]>([]);
+  const [linkedAccounts, setLinkedAccounts] = useState<
+    LinkedAccountRepresentation[]
+  >([]);
+  const [unlinkedAccounts, setUninkedAccounts] = useState<
+    LinkedAccountRepresentation[]
+  >([]);
 
+  const [paramsUnlinked, setParamsUnlinked] = useState<Record<string, any>>({
```

Comment:
Let's avoid using `any` as a type, please create a new type for the parameters that matches this data structure.